File: app/src/main/java/com/webianks/hatkemessenger/services/HeadlessSmsSendService.java

package com.webianks.hatkemessenger.services;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;

import androidx.annotation.Nullable;

/**
 * Created by R Ankit on 24-12-2016.
 */
public class HeadlessSmsSendService extends Service {
    @Nullable
    @Override
    public IBinder onBind(Intent intent) {
        return null;
    }
}

File: app/src/main/java/com/webianks/hatkemessenger/services/HeadlessSmsSendService.kt
package com.webianks.hatkemessenger.services

import android.app.Service
import android.content.Intent
import android.os.IBinder

/**
 * Created by R Ankit on 24-12-2016.
 */
class HeadlessSmsSendService : Service() {
    override fun onBind(intent: Intent): IBinder? {
        return null
    }
}